README.md

```diff
@@ -68,14 +68,22 @@ An object with the same interface as `THREE.Mesh` but used to evaluate CSG opera
 > 
 > It is recommended to remove groups from a geometry before creating a brush if multi-material support is not required.
 
-<!--
 ## Operation
 
 _extends Brush_
 
-TODO
+This is an extension of the `Brush` class. With this class is possible to create a mesh and pass the CSG Operation constant to define which operation computes on the mesh. If you don't choose an operation, the default is `ADDITION`. The result is a Mesh whose effect is defined from the operation selected. You can see how it works in the _hierarchy_ example.
```

Comment:
We'll want to document the local member of `.operation`, as well:

````md
### .operation

```js
operation = ADDITION : CSGOperation
```
````

We'll also want to document the "insertBefore" and "insertAfter" functions.